gen3/bin/gitops.sh

```diff
@@ -291,9 +291,15 @@ gen3_gitops_sync() {
   if g3kubectl get configmap manifest-versions; then
     oldJson=$(g3kubectl get configmap manifest-versions -o=json | jq ".data")
   fi
-  newJson=$(g3k_config_lookup ".versions")
   echo "old JSON is: $oldJson"
-  echo "new JSON is: $newJson"
+  newJson=$(g3k_config_lookup ".versions")
```

Comment:
I think this update will make sure it fails if any part of the manifest is invalid json, not just the "versions" section, if that's the case could you update the PR description

Reply:
yes! Thanks so much Pauline :D Just updated the description